Library/OcAppleKernelLib/CommonPatches.c

```diff
@@ -242,6 +248,11 @@ PatchAppleXcpmCfgLock (
     return EFI_SUCCESS;
   }
 
+  //
+  // This is a kernel patch, so Patcher cannot be NULL.
+  //
+  ASSERT (Patcher != NULL);
```

Comment:
Makes little sense to move it here. If it is a kernel patch, then Patcher must not be NULL even on not supported releases.

Reply:
[Fixed.](https://github.com/acidanthera/OpenCorePkg/pull/359/commits/9d76078058c197100bb369e3849387a18d51a6e6)